### DISCUSSION TITLE
review-init -wからこの設定を取得するために使用する正しい値を知る方法は？

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1842] [Q&A] [answered]
review-init -wからこの設定を取得するために使用する正しい値を知る方法は？

例：
ノド=18.75= mm
小口=18.75mm
天=15.80mm
地=32.22mm

でそれを使用する方法？
texdocumentclass：["review-jsbook"、 "media = print、paper = a5"]

![image](https://user-images.githubusercontent.com/49851974/181197922-0f198cfc-a684-4a5e-9a39-b3e87e39ef9c.png)

**Comment by kmuto** [comment, marked as answer]
天地ノドなどを調整して「作成」すると、そのプロジェクトのconfig.ymlのtexdocumentclassに書き出されるので、それを本番のプロジェクトのほうに転記します。

もし何もそれらしいパラメータ変化がないのであれば、それはデフォルト値だからです。

**Reply by jasondee1992** [reply]
Thank you kmuto san.
